src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 3 && days <= 6) {
```

Comment:
here, days >= 3 is always `true`, because the previous condition is already omitted
also, what's 6? let's use given values only